<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherReleaseUI</class>
 <widget class="QDialog" name="TeacherReleaseUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>华工教务</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="coursenameLabel">
      <property name="text">
       <string>课程名称</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="coursenameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="coursetimeLabel">
      <property name="text">
       <string>上课时间</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="coursetimeLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="courseroomLabel">
      <property name="text">
       <string>上课地点</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="coursemaxLabel">
      <property name="text">
       <string>课程容量</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="coursemaxLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="courseroomLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发布课程</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
